components/modal/index.jsx

```diff
@@ -188,8 +188,13 @@ class Modal extends React.Component {
 			if (this.state.isClosing) {
 				// console.log("CLOSING: ');
 				if (!this.isUnmounting) {
-					const el = ReactDOM.findDOMNode(this).parentNode; // eslint-disable-line react/no-find-dom-node
-					if (el && el.getAttribute('data-slds-modal')) {
+					const el = ReactDOM.findDOMNode(this); // eslint-disable-line react/no-find-dom-node
```

Comment:
Is it possible to not use `findDOMNode` and use a ref here? Modal, App Launcher and Dropdown are the last components to use `findDOMNode`. [More info](https://github.com/yannickcr/eslint-plugin-react/issues/678) on the topic.

Reply:
Nevermind. I looked into it. About three years ago (maybe before Portals existed in React?), you could use `modal/trigger.jsx` to create a modal in a portal. I think the path forward should be to keep this in until we can remove it. It doesn't seem to be run with `Modal/index.js`. I'll push a comment on why it's here. We should probably deprecate the trigger/manager modal paradigm if it was just created to create portals before portals existed.

@ivanbogdanov @donnieberg Is the above statement accurate? Was the "Modal Manager" created as a way to get portals, before they existed in React. This code is before my time: `v0.0.25-27` circa 2015.

Reply:
React Modal does it's own portal now:

![screen shot 2018-05-23 at 3 59 12 pm](https://user-images.githubusercontent.com/1290832/40453405-4ebc2242-5ea2-11e8-8bc9-dfdf940eeff0.png)